fundamental-concepts/data-structures-stdlib/0_intro.md

```diff
@@ -0,0 +1,14 @@
+Choosing the right types for your data will make a big difference for the efficiency, complexity and
+clarity of your code. Furthermore, data structures can encode important invariants of your code.
+Examples are non-empty lists or optional fields. 
+
+DAML excels when it comes to modeling data. Its standard library provides you with a rich
+collection of common data containers and efficient algorithms.
+
+In this scenario you will learn about 
+
+  - basic data types of DAML
+  - how to import the standard library or your own modules
```

Comment:
```suggestion
  - how to import your own modules or standard library
```
That's how it's ordered (own comes first)

Reply:
done